index.rst

```diff
@@ -332,12 +307,13 @@ For example, we want cutouts to be done in a long-running process that pays the
 Output
 ~~~~~~
 
-The output cutout should be a FITS image stored in the output Butler collection.
+The output cutout should be a FITS image stored in the provided GCS bucket.
+It may be accompanied by a metadata file.
```

Comment:
If this is desired for the first version, I think we can get it done pretty easily, but there's no follow-up ticket for it at present (commenting because I see a few other things that are explicitly marked as "not supported in the initial implementation" that I have created tickets for).

Reply:
I have no preference on whether to do this for the first version, or about when we should do this. I just wanted to note that this was possible and make it clear that the design wasn't ruling this out by, for example, making it look like the backend was required to create only a single file. In terms of creating tickets, I'll defer to Frossie and GPDF on that.